Library/Homebrew/cli/args.rb

```diff
@@ -165,10 +186,12 @@ def cli_args
         @cli_args.freeze
       end
 
-      def respond_to_missing?(method_name, *)
+      sig { params(method_name: Symbol, _include_private: T::Boolean).returns(T::Boolean) }
+      def respond_to_missing?(method_name, _include_private = false)
```

Comment:
Not sure why it didn't error here, but it's now erroring on `master`:

https://github.com/Homebrew/brew/actions/runs/10469292764/job/28991992536#step:7:14

Reply:
#18097